src/FSharp.Core/map.fs

```diff
@@ -748,206 +905,277 @@ type Map<[<EqualityConditionalOn>]'Key, [<EqualityConditionalOn; ComparisonCondi
 
         member m.Count = m.Count
 
-    interface System.IComparable with 
-        member m.CompareTo(obj: obj) = 
-            match obj with 
-            | :? Map<'Key, 'Value>  as m2->
-                Seq.compareWith 
-                   (fun (kvp1 : KeyValuePair<_, _>) (kvp2 : KeyValuePair<_, _>)-> 
-                       let c = comparer.Compare(kvp1.Key, kvp2.Key) in 
-                       if c <> 0 then c else Unchecked.compare kvp1.Value kvp2.Value)
-                   m m2 
-            | _ -> 
-                invalidArg "obj" (SR.GetString(SR.notComparable))
+    interface System.IComparable with
+        member m.CompareTo(obj: obj) =
+            match obj with
+            | :? Map<'Key, 'Value> as m2 ->
+                Seq.compareWith
+                    (fun (kvp1: KeyValuePair<_, _>) (kvp2: KeyValuePair<_, _>) ->
+                        let c = comparer.Compare(kvp1.Key, kvp2.Key) in
+
+                        if c <> 0 then
+                            c
+                        else
+                            Unchecked.compare kvp1.Value kvp2.Value)
+                    m
+                    m2
+            | _ -> invalidArg "obj" (SR.GetString(SR.notComparable))
 
     interface IReadOnlyCollection<KeyValuePair<'Key, 'Value>> with
         member m.Count = m.Count
 
     interface IReadOnlyDictionary<'Key, 'Value> with
 
-        member m.Item with get key = m.[key]
+        member m.Item
+            with get key = m.[key]
 
         member m.Keys = m.Keys :> IEnumerable<'Key>
 
-        member m.TryGetValue(key, value: byref<'Value>) = m.TryGetValue(key, &value) 
+        member m.TryGetValue(key, value: byref<'Value>) = m.TryGetValue(key, &value)
 
         member m.Values = m.Values :> IEnumerable<'Value>
 
         member m.ContainsKey key = m.ContainsKey key
 
-    override x.ToString() = 
-        match List.ofSeq (Seq.truncate 4 x) with 
+    override x.ToString() =
+        match List.ofSeq (Seq.truncate 4 x) with
         | [] -> "map []"
-        | [KeyValue h1] ->
+        | [ KeyValue h1 ] ->
             let txt1 = LanguagePrimitives.anyToStringShowingNull h1
-            StringBuilder().Append("map [").Append(txt1).Append("]").ToString()
-        | [KeyValue h1; KeyValue h2] ->
+
+            StringBuilder()
+                .Append("map [")
+                .Append(txt1)
+                .Append("]")
+                .ToString()
+        | [ KeyValue h1; KeyValue h2 ] ->
             let txt1 = LanguagePrimitives.anyToStringShowingNull h1
             let txt2 = LanguagePrimitives.anyToStringShowingNull h2
-            StringBuilder().Append("map [").Append(txt1).Append("; ").Append(txt2).Append("]").ToString()
-        | [KeyValue h1; KeyValue h2; KeyValue h3] ->
+
+            StringBuilder()
+                .Append("map [")
+                .Append(txt1)
+                .Append("; ")
+                .Append(txt2)
+                .Append("]")
+                .ToString()
```

Comment:
https://github.com/dotnet/fsharp/pull/13156/files#diff-0947e2727d6bad8cd0ac4122f5314bb5b04e337393075bc4b5ef143b17fcbd5bR8 may help this (setting `fsharp_max_dot_get_expression_width` wider)